src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,15 @@
-import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList = ({ ...moviesObject }) => (
+  <div className="movies">
+    {moviesObject.movies.map(
```

Comment:
You should receive array of movies 
```suggestion
    {movies.map(
```

Reply:
I decomposed movies.